pkg/loops/loop.go

```diff
@@ -112,12 +112,6 @@ func (r *Loop) Parse(data string, args ...string) (int, error) {
 	index := 0
 
 	for rule := segRules.Get(index); rule != nil; rule = segRules.Get(index) {
-
-		cnt := 1
-		if rule.RepeatCount > cnt {
-			cnt = rule.RepeatCount
-		}
```

Comment:
Shouldn't `cnt` still be assigned to `rule.RepeatCount`?

Reply:
if RepeatCount is omit, Repeat() functions will return default value 1

Reply:
Ah okay.